### PR TITLE
account tile moved underline to call to action text

### PR DESCRIPTION
**Pull request by jonedollarz**

### CODE REVIEW

**Comment by stephiepea** [comment]
:shipit:

File: src/tile-account/_tile-account.scss [hunk 1]
```diff
@@ -17,6 +17,14 @@
     }
   }
 
+  &:hover .tile-title {
```
Comment (david-nicholson) [inline src/tile-account/_tile-account.scss]: Probably should group the <code>&:hover</code>s into 1...

Comment (aswitalski) [inline src/tile-account/_tile-account.scss]: Agreed.

**Comment by david-nicholson** [comment]
:shipit:

**Comment by aswitalski** [comment]
:shipit:

**Pull request merged**